lib/fizz_buzz/game.rb

```diff
@@ -5,14 +5,7 @@ def display_plain_number(first_number, last_number)
       if !validate_numbers(first_number, last_number)
         "Not accepted numbers"
       else
-        next_number = first_number
-        number_series = ""
-        while next_number <= last_number
-            number_series += "#{next_number}"
-            number_series += (next_number != last_number ?  "," : "")
-            next_number += 1
-        end
-        number_series
+        (first_number..last_number).map(&:to_s).join(",")
```

Comment:
you don't need to map them to strings. The join will do it for you

Reply:
got it

Reply:
I removed the mapping to strings.
In the newer version, (first..last) is a range so i casted it to array then use join for it to work